test/expiring-todo-comments.js

```diff
@@ -110,6 +110,22 @@ ruleTester.run('expiring-todo-comments', rule, {
 		{
 			code: '// TODO [but [it will]] [fallback] [[[ to the default ]]] rule [[',
 			errors: []
+		},
+		{
+			code: '// TODO ISSUE-123 fix later',
```

Comment:
I thought those comments requires `:` inside? Can you remove options  make sure  they  are  reported , then add options back.

Reply:
Not sure if I understand what you mean.

When I remove `ignore: ['ISSUE-\\d+']` the comment is reported as "Unexpected 'todo' comment."

Does that answer it?